cmd/skupper/skupper.go

```diff
@@ -852,6 +715,17 @@ func init() {
 	cmdVersion := NewCmdVersion(newClient)
 	cmdDebugDump := NewCmdDebugDump(newClient)
 
+	//backwards compatible commands tunning
+	cmdListConnectors := NewCmdListConnectors(newClient) //listconnectors just keeped
+	cmdListConnectors.Hidden = true
+
+	cmdConnect := NewCmdConnect(newClient)
+	cmdConnect.Hidden = true
```

Comment:
Suggest adding cmd.Deprecated string for these command suggesting use of new 'skupper link [command]'